gradle.properties

```diff
@@ -20,11 +20,11 @@ buildKotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds
 remoteRoot=konan_tests
 kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_Compiler),number:1.3.40-dev-2423,branch:default:true,pinned:true/artifacts/content/maven
 kotlinVersion=1.3.40-dev-2423
-testKotlinVersion=1.3.40-dev-2423
+testKotlinVersion=1.3.40-dev-2454
 # See https://teamcity.jetbrains.com/project.html?projectId=Kotlin_KotlinNativeShared&tab=projectOverview
 sharedRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_KotlinNativeShared_BuildAndTest),number:1.0-dev-49,branch:default:true,pinned:true/artifacts/content/maven
 sharedVersion=1.0-dev-49
 konanVersion=1.3.0
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 org.gradle.workers.max=4
-#kotlinProjectPath=/Users/jetbrains/kotlin-native/kotlin
+kotlinProjectPath=//Users/abduqodiri.qurbonzoda/kotlin
```

Comment:
No.

Reply:
There is no successful build of master branch containing necessary api changes to update `kotlinVersion`.
This file will be modified properly when circumstances permit.

Reply:
Fixed in PR #2973